langserver/handler.go

```diff
@@ -159,6 +160,7 @@ func (h *langHandler) linter() {
 			&PublishDiagnosticsParams{
 				URI:         uri,
 				Diagnostics: diagnostics,
+				Version:     h.files[uri].Version,
```

Comment:
I'm not sure but this have to check nil ?

Reply:
thank you for your review!

> this have to check nil ?

I think no need. `h.lint()` guards it.
but this may be implicit and not best.
the following is the explicit impl.

```diff
diff --git a/langserver/handler.go b/langserver/handler.go
index 0c524b0..5d4d3fa 100644
--- a/langserver/handler.go
+++ b/langserver/handler.go
@@ -154,13 +154,20 @@ func (h *langHandler) linter() {
 			h.logger.Println(err)
 			continue
 		}
+
+		f, ok := h.files[uri]
+		if !ok {
+			h.logger.Println(fmt.Errorf("document not found: %v", uri))
+			continue
+		}
+
 		h.conn.Notify(
 			context.Background(),
 			"textDocument/publishDiagnostics",
 			&PublishDiagnosticsParams{
 				URI:         uri,
 				Diagnostics: diagnostics,
-				Version:     h.files[uri].Version,
+				Version:     f.Version,
 			})
 	}
 }
```

Reply:
Thanks your confirmation.